### DISCUSSION TITLE
Anyone tested NanoCul on Raspi with wmbusmeters?

### DISCUSSION THREAD

**Discussion by volkerdeHaas** [wmbusmeters/wmbusmeters#1244] [Q&A]
I finally managed to get my NanoCUL running with wmbusmeters on a Raspi. But, alas, the nanoCUL device vanishes after several seconds! I could just fetch a single telegram or sometimes no telegram before ttyusb0 vanishes magically from my lsusb list. I need to unplug and replug the USB connector to get it back again. This issue (vanishing ttyusb0 after several seconds) is also present without starting wmbusmeters. It looks like the NanCUL shuts off its USB connection. Has anyone had similar effects? Is there any solution? I have already re-flashed the NanoCUL but it does not help.

**Comment by tomtomtom2** [comment]
Did you try a dedicated power supply for th pi?

**Reply by volkerdeHaas** [reply]
Certainly. I gave up with nanoCul and I'm successfully using a different USB Stick which works fine for me.

**Reply by tomtomtom2** [reply]
Which USB Stick are u using?

**Reply by volkerdeHaas** [reply]
https://shop.imst.de/wireless-solutions/usb-radio-products/10/im871a-usb-wireless-m-bus-usb-adapter-868-mhz
